pkg/kwokctl/runtime/compose/cluster.go

```diff
@@ -46,13 +46,37 @@ import (
 // Cluster is a implementation of Runtime for docker-compose.
 type Cluster struct {
 	*runtime.Cluster
+
+	driver string
 }
 
-// NewCluster creates a new Runtime for docker-compose.
-func NewCluster(name, workdir string) (runtime.Runtime, error) {
-	return &Cluster{
+var minNerdctlVersion = version.NewVersion(1, 2, 0)
+
+// NewNerdctlCluster creates a new Runtime for nerdctl compose.
+func NewNerdctlCluster(name, workdir string) (runtime.Runtime, error) {
+	cluster := &Cluster{
 		Cluster: runtime.NewCluster(name, workdir),
-	}, nil
+		driver:  consts.RuntimeTypeNerdctl,
+	}
+
+	v, err := version.ParseFromBinary(context.Background(), cluster.driver)
+	if err != nil {
+		return nil, err
+	}
+	if v.LT(minNerdctlVersion) {
+		return nil, fmt.Errorf("nerdctl: minimal supported version is %s", &minNerdctlVersion)
```

Comment:
Why only support the 1.2.0 released yesterday, this approach is not appropriate.

Reply:
We need to maintain compatibility with versions prior to 1.2.0

Reply:
reverted changes and drop the limitation

Reply:
This work is nearly complete in another PR kubernetes-sigs/kwok#249